app/Repositories/Summit/DoctrineSummitEventRepository.php

```diff
@@ -523,6 +525,83 @@ protected function getCustomFilterMappings(int $current_member_id, int $current_
                     WHERE p4:i.id = p.id AND mut2:i.id :operator :value
                 )'
             ),
+            'review_status' => new DoctrineSwitchFilterMapping
+            (
+                [
+                    Presentation::ReviewStatusNoSubmitted => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusNoSubmitted,
+                        "selp IS NULL OR (p.status <> 'Received' AND p.status <> 'Accepted')"
+                    ),
+                    Presentation::ReviewStatusReceived => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusReceived,
+                        "p.status = 'Received' AND e.published = 0 AND
+                            selp IS NOT NULL AND ( 
+                                (selp.submission_lock_down_presentation_status_date IS NULL OR selp.submission_lock_down_presentation_status_date <= '$now_utc') AND 
+                                selp.submission_begin_date <= '$now_utc' AND 
```

Comment:
dont use now utc like this use a DATABASE TIMESTAMP

Reply:
done